<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="HSV">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>470</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>HSV Slicing</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>821</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Batch                                                                                                                                                     ...</string>
    </property>
   </widget>
   <widget class="QWidget" name="ImgWidget" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>-94</x>
       <y>700</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Value (Min)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>-80</x>
       <y>620</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Hue (Min)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>-124</x>
       <y>660</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Saturation (Min)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../home/tamil/Downloads/0.png</pixmap>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>590</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="S1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>560</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="V2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>600</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>520</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hue (Max)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>600</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Value (Min)</string>
    </property>
   </widget>
   <widget class="QSlider" name="H2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>520</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>600</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Value (Max)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>550</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="V1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>600</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>560</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>    Saturation (Max)</string>
    </property>
   </widget>
   <widget class="QSlider" name="H1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>560</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturation (Min)</string>
    </property>
   </widget>
   <widget class="QSlider" name="S2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>560</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hue (Min)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_6">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>590</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>510</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_7">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>550</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>340</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 44, 29);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QListWidget" name="imgs_list">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>311</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <zorder>save</zorder>
   <zorder>HSV</zorder>
   <zorder>startImage</zorder>
   <zorder>ImgWidget</zorder>
   <zorder>label_8</zorder>
   <zorder>spinBox_3</zorder>
   <zorder>S1</zorder>
   <zorder>V2</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>H2</zorder>
   <zorder>label_7</zorder>
   <zorder>spinBox_2</zorder>
   <zorder>V1</zorder>
   <zorder>label_12</zorder>
   <zorder>H1</zorder>
   <zorder>label_13</zorder>
   <zorder>S2</zorder>
   <zorder>spinBox</zorder>
   <zorder>label_2</zorder>
   <zorder>spinBox_6</zorder>
   <zorder>spinBox_4</zorder>
   <zorder>spinBox_7</zorder>
   <zorder>close_button</zorder>
   <zorder>imgs_list</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_image">
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="actionOpen_video">
   <property name="text">
    <string>Open video</string>
   </property>
  </action>
  <action name="actionStart_Webcam">
   <property name="text">
    <string>Start Webcam</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>H1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>S1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>805</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>800</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>V1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>845</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>840</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>H2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>1110</x>
     <y>760</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>S2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>805</y>
    </hint>
    <hint type="destinationlabel">
     <x>1110</x>
     <y>800</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>V2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>926</x>
     <y>844</y>
    </hint>
    <hint type="destinationlabel">
     <x>1110</x>
     <y>840</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>H1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>731</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>725</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>S1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>763</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>764</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>V1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>803</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>806</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>H2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1127</x>
     <y>723</y>
    </hint>
    <hint type="destinationlabel">
     <x>1065</x>
     <y>727</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>S2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1122</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>1039</x>
     <y>766</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>V2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1131</x>
     <y>796</y>
    </hint>
    <hint type="destinationlabel">
     <x>1020</x>
     <y>810</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
